docs/docs/recipes/organizations/organization-invitations.mdx

```diff
@@ -0,0 +1,128 @@
+  - `invite:member` - Invite members to the organization.
+
+## Configure your email connector
+
+Since invitations are sent via email, ensure your email connector is properly configured. To send invitations, you need to configure a newly introduced email template usage type - `OrganizationInvitation`.
+
+:::note
+Logto Cloud's built-in "Logto email service" does not support the `OrganizationInvitation` usage type at the moment. Instead, you need to configure your email connector (e.g. Sendgrid) and set up the `OrganizationInvitation` template.
+:::
+
+A sample email template for the `OrganizationInvitation` usage type is shown below:
+
+```json
+{
+  "subject": "Welcome to my organization",
+  "content": "<p>Join my organization by this <a href=\"{{link}}\" target=\"_blank\">link</a>.</p>",
+  "usageType": "OrganizationInvitation",
+  "type": "text/html"
+}
+```
+
+The `{{link}}` placeholder in the email content will be replaced with the actual invitation link when sending the email. In this guide, let's say it would be `https://your-app.com/invitation/accept/your-invitation-id`.
+
+You can check out the [Configure popular email service](/docs/recipes/configure-connectors/email-connector/configure-popular-email-service/) guide for more details on setting up email connectors.
+
+## Handle invitations with Logto management API
+
+:::note
+If you haven't set up the Logto management API yet, check out [Interact with Management API](/docs/recipes/interact-with-management-api/) for details.
+:::
+
+We've provided a set of invitation-related management APIs in the organizations feature. With these APIs, you can:
+
+- `POST /api/organization-invitations` create an organization invitation with an assigned organization role.
+- `POST /api/organization-invitations/{id}/message` send the organization invitation to invitee via email.
+
+  Note: This API payload supports a `link` property, you can compose your invitation link based on the invitation ID. For example:
+
+  ```json
+  {
+    "link": "https://your-app.com/invitation/accept/your-invitation-id"
+  }
+  ```
+
+  Accordingly, you need to implement a landing page when your invitee navigates through the invitation link to your application.
+
+- `GET /api/organization-invitations` & `GET /api/organization-invitations/{id}` get all your invitations or a specific one by ID.
+
+  On your landing page, use these APIs to list all invitations or details of an invitation that a user has received.
+
+- `PUT /api/organization-invitations/{id}/status` accept or reject the invitation by updating the invitation status.
+
+  Use this API to handle the user's response to the invitation.
+
+Please be noted that all the APIs listed above requires a valid "organization access token". Check this [guide](/docs/recipes/organizations/integration/#step-2-fetch-the-organization-token) to learn how to obtain the organization access token.
+
+## Use RBAC to manage user permissions
```

Comment:
org level RBAC?
Since we are using org role here, better to specify the concept and avoid confusing

Reply:
anyways, updated